definitions/npm/@reach/router_v1.1.x/flow_v0.63.x-v0.103.x/router_v1.1.x.js

```diff
@@ -130,4 +137,6 @@ declare module '@reach/router' {
   declare export function isRedirect(error: any): $Exact<{ uri: string }>;
 
   declare export function redirectTo(uri: string): void;
+
+  declare export function useLocation(): $PropertyType<UseLocation, 'location'>;
```

Comment:
Well, there's no `location` key in the `UseLocation` type you added.

Reply:
@pascalduez resolved with current [pull request](https://github.com/flow-typed/flow-typed/pull/3937/)